tests/Unit/IO/Test_H5.py

```diff
@@ -0,0 +1,89 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from spectre import DataStructures
+from spectre.IO.H5 import IOH5File as  spectre_h5
+import unittest
+import numpy as np
+import os
+
+
+class TestIOH5File(unittest.TestCase):
+    # Test Fixtures
+    def setUp(self):
+        self.file_name = "./pythontest.h5"
+        self.data_1 = [1.0, 3.5]
+        self.data_1_array = np.array(self.data_1)
+        self.data_2 = [3.0, 10.3]
+        self.data_2_array = np.array(self.data_2)
+        self.cd = os.getcwd()
+        if "pythontest.h5" in os.listdir(self.cd):
+            os.remove(self.file_name)
+
+    # Test whether an H5 file is created correctly, delete a file with the name
+    # 'pythontest.h5' if it already exists.
+    def test_name(self):
+        file_spec = spectre_h5.H5File(self.file_name, 1)
```

Comment:
What do you mean with the `_spec` suffix? I think just remove it

Reply:
@nilsleiffischer each test needs a fresh copy of the file,  so as it stands it needs to be deleted at the end of each test so that when `spectre_h5.H5File` is called in the next test it creates new file, with no `dat` files in it that may conflict with the previous tests.

Reply:
I guess this was originally to avoid confusion with the Built-in function `file()`, it doesn't seem to cause any problems if `file_spec` is changed to `file` but it may be confusing if people call `help(file)` for example.

Reply:
ok, how about h5_file then?

Reply:
Will do!